deploy/addon/mongodb/managing-MongoDB.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Managing your MongoDB"
+date: 2023-06-20T15:01:20+02:00
+position: 3
+shortdesc: Managing your MongoDB
+tags:
+- monitoring
+- document
+- nosql
+- database
+- managed service
+---
+
+## Can I use [Mongo Ops Manager](https://www.mongodb.com/products/ops-manager) on Clever Cloud?
+
+To be able to use Mongo Ops Manager, you'll need a valid MongoDB Enterprise Advanced subscription and to deploy a [Linux version of their manager solution](https://www.mongodb.com/try/download/ops-manager). If you don't have purchased any license from MongoDB and you are using the Community version, you might be looking for a similar service for your databases.
```

Comment:
HTML `id`s are generated automatically, and having URLs inside the `<h2>`s can possibly cause long and unreadable `id`s. We can just move the link to the paragraph below.

```suggestion
## Can I use Mongo Ops Manager on Clever Cloud?

To be able to use [Mongo Ops Manager](https://www.mongodb.com/products/ops-manager), you'll need a valid MongoDB Enterprise Advanced subscription and to deploy a [Linux version of their manager solution](https://www.mongodb.com/try/download/ops-manager). If you haven't purchased any license from MongoDB and you are using the Community version, you might be looking for a similar service for your databases.
```